packages/react-components/components/MoneyAmount.tsx

```diff
@@ -48,4 +50,12 @@ const style = StyleSheet.create({
     fontSize: 24,
     color: colors.darkSecondary,
   },
+  currencyCode: {
+    ...fontStyles.regular,
+    fontSize: 16,
```

Comment:
The `body` font style already specifies this size and font family, consider using that instead.